tests/experimental/delete_bykey.phpt

```diff
@@ -21,7 +21,7 @@ var_dump($m->deleteByKey('keffe', ''));
 echo $m->getResultMessage(), "\n";
 var_dump($m->deleteByKey('', 'keffe'));
 echo $m->getResultMessage(), "\n";
-var_dump($m->deleteByKey('keffe', '���as�� �a�sd�f asdf'));
+var_dump($m->deleteByKey('keffe', '���as�� �a�sd�f asdf')); # no spaces allowed
```

Comment:
I know it seems like I just changed these to pass, but it really looks like this is the expected result. strchr is ran on the keys everywhere to check for spaces ...